src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 .stars__star:nth-child(-n+5),
```

Comment:
u can set this image for all stars, and add yellow only for needed, it will work the same way